test/support/test_config.h

```diff
@@ -180,6 +180,7 @@
 #define _PSTL_ICC_TEST_COMPLEX_POW_SCALAR_COMPLEX_PASS_BROKEN_TEST_EDGES  _PSTL_TEST_COMPLEX_OP_BROKEN_GLIBCXX
 #define _PSTL_ICC_TEST_COMPLEX_NORM_MINUS_INF_NAN_BROKEN_TEST_EDGES       _PSTL_TEST_COMPLEX_OP_BROKEN_GLIBCXX
 #define _PSTL_ICC_TEST_COMPLEX_POLAR_BROKEN_TEST_EDGES                    _PSTL_TEST_COMPLEX_OP_BROKEN_GLIBCXX
+#define _PSTL_ICC_TEST_COMPLEX_EXP_BROKEN_TEST_EDGES                     (__INTEL_LLVM_COMPILER > 0 && 20220819 < __INTEL_LLVM_COMPILER && __INTEL_LLVM_COMPILER <= _PSTL_TEST_LATEST_INTEL_LLVM_COMPILER)

```

Comment:
Should this be `20220819 < __GLIBCXX__` ?

Reply:
No: in the previous compiler version without these new errors and in the some current compiler version with the errors the state of `__GLIBCXX__ ` is the same:
```C++
_GLIBCXX__ = 20220819
```